dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+interval = "15s"
+# The Prometheus Pushgateway address.
+address = ""
+
+[rocksdb]
+# The maximum number of concurrent background jobs.
+max-background-jobs = 32
+
+[rocksdb.defaultcf]
+# Amount of data to build up in memory before flushing data to the disk.
+write-buffer-size = "1GB"
+# The maximum number of write buffers that are built up in memory.
+max-write-buffer-number = 8
+
+# The compression algorithms used in different levels.
+# The algorithm at level-0 is used to compress KV data.
+# The algorithm at level-6 is used to compress SST files.
+# The algorithms at level-1 to level-5 are unused for now.
+compression-per-level = ["lz4", "no", "no", "no", "no", "no", "lz4"]
+
+[rocksdb.writecf]
+# (same as above)
+compression-per-level = ["lz4", "no", "no", "no", "no", "no", "lz4"]
+
+[import]
+# The directory to store engine files.
+import-dir = "/mnt/ssd/data.import/"
+# Number of threads to handle RPC requests.
+num-threads = 16
+# Number of concurrent import jobs.
+num-import-jobs = 24
+# Maximum duration to prepare Regions.
+#max-prepare-duration = "5m"
+# Split Regions into this size according to the importing data.
+#region-split-size = "512MB"
+# Stream channel window size. The stream will be blocked on channel full.
+#stream-channel-window = 128
+# Maximum number of open engines.
+max-open-engines = 8
+# Maximum upload speed (bytes per second) from Importer to TiKV.
+# upload-speed-limit = "512MB"
+# Minimum ratio of available space on the target store: `store_available_space`/`store_capacity`.
+# Importer pauses uploading SST if the availability ratio of the target store is less than this
+# value, to allow enough time for PD to balance Regions.
+min-available-ratio = 0.05
+```
+
+## Command line parameters
+
+### Usage of `tidb-lightning`
+
+| Parameter | Explanation | Overrides setting |
+|:----|:----|:----|
+| --config *file* | Reads global configuration from *file*. If not specified, the default configuration would be used. | |
+| -V | Prints program version | |
+| -d *directory* | Directory of the data dump to read from | `mydumper.data-source-dir` |
+| -L *level* | Log level: debug, info, warn, error, fatal (default = info) | `lightning.log-level` |
+| --log-file *file* | Log file path | `lightning.log-file` |
+| --status-addr *ip:port* | Listening address of the TiDB Lightning server | `lightning.status-port` |
+| --importer *host:port* | Address of TiKV Importer | `tikv-importer.addr` |
+| --pd-urls *host:port* | PD endpoint address | `tidb.pd-addr` |
+| --tidb-host *host* | TiDB server host | `tidb.host` |
+| --tidb-port *port* | TiDB server port (default = 4000) | `tidb.port` |
+| --tidb-status *port* | TiDB status port (default = 10080) | `tidb.status-port` |
+| --tidb-user *user* | User name to connect to TiDB | `tidb.user` |
+
+## Usage of `tidb-lightning-ctl`
+
+All parameters of `tidb-lightning` can be used in `tidb-lightning-ctl`. Additionally, the following parameter should be supplied to execute the command.
```

Comment:
This sentence is kind of confusing to me. Could you make it clearer? Thanks!

`Additionally, the following parameter should be supplied to execute the command.`

- What's the logic with the former sentence?
- What command?
- To execute the command, users must configure all parameters listed below?